lua/lvim/plugins.lua

```diff
@@ -76,16 +76,14 @@ return {
   -- Install nvim-cmp, and buffer source as a dependency
   {
     "hrsh7th/nvim-cmp",
+    event = "BufEnter",
     commit = commit.nvim_cmp,
     config = function()
       require("lvim.core.cmp").setup()
     end,
-    run = function()
-      -- cmp's config requires cmp to be installed to run the first time
-      if not lvim.builtin.cmp then
-        require("lvim.core.cmp").config()
-      end
-    end,
+    requires = {
+      "L3MON4D3/LuaSnip",
+    },
```

Comment:
This combination seems to work, along with assigning an empty config regardless if cmp is available or not.
I had wanted to use "BufRead" or "InsertEnter" but I couldn't get it either to play well with Telescope which will be triggered from Dashboard.

@chase, what do you think?

Reply:
I guess I'm wondering why we're lazy loading nvim-cmp anyway, it just delays having it load in the cache which brings a much more significant speedup.

Reply:
I basically don't want it to try to run `setup()` while in headless mode, since it was conflicting with Packer for some reason. Otherwise, it's not really that useful.

I'll try to remove it once more to see if it's somehow sorted now.